compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -108,6 +106,16 @@ class ComposeScene internal constructor(
         invalidate,
     )
 
+    private val childScenes = mutableSetOf<ComposeScene>()
```

Comment:
Not sure about naming here.
1. child -> children
2. "child" implies ownership, but it's not

Reply:
Not sure why `childScenes` implies ownership ;-)